packages/ckeditor5-ckbox/lang/contexts.json

```diff
@@ -1,5 +1,6 @@
 {
   "Open file manager": "Toolbar button tooltip for opening a file browser that allows inserting an image or a file to the editor.",
   "Cannot determine a category for the uploaded file.": "A message is displayed when CKEditor 5 cannot associate an image with any of the categories defined in CKBox while uploading an asset.",
-  "Cannot access default workspace.": "A message is displayed when the user is not authorised to access CKBox workspace configured as default one."
```

Comment:
Inconsistencies: "authorized" vs. "authorised".
Missing articles: "the CKBox workspace", "the default one".

Reply:
This is part of https://github.com/ckeditor/ckeditor5/pull/15209, won't be addressing it here.

@pszczesniak have a look.